README.md

```diff
@@ -140,43 +140,64 @@ GraphQL-AnyCable uses [anyway_config] to configure itself. There are several pos
 
 And any other way provided by [anyway_config]. Check its documentation!
 
+## Broadcasting
+
+```ruby
+class MySchema < GraphQL::Schema
+  use GraphQL::Execution::Interpreter
```

Comment:
Not sure if it worth mentioning in the README, but Interpreter and AST are turned on by default starting from graphql-ruby 1.12.4

Reply:
Subscription broadcast feature exists starting from 1.11 and it requires Interpreter (otherwise GraphQL-Ruby will throw error), so yes, probably I need to mention it in the README

Reply:
Added notes to README and CHANGELOG